docs/tutorials/11_GMPHDTutorial.py

```diff
@@ -0,0 +1,577 @@
+    axL.view_init(elev=30, azim=-80)
+    axL.set_zlim(0, 0.3)
+
+    # Initialize the variables
+    num_gaussians = len(all_gaussians[i]) # number of functions in this mixture
+    weights = []  # weights of each Gaussian. This is analogous to the probability of its existence
+    means = []    # means of each Gaussian. This is equal to the x and y of its state vector
+    sigmas = []   # standard deviation of each Gaussian.
+
+    # Fill the lists of weights, means, and standard deviations
+    for state in all_gaussians[i]:
+        weights.append(state.weight)
+        means.append([state.state_vector[0], state.state_vector[2]])
+        sigmas.append([state.covar[0][0], state.covar[1][1]])
+    means = np.array(means)
+    sigmas = np.array(sigmas)
+
+    # Generate the z values over the space and plot on the left axis
+    zarray[:,:,i] = get_mixture_density(x, y, weights, means, sigmas)
+    sf = axL.plot_surface(x, y, zarray[:,:,i], cmap=cm.RdBu, linewidth=0, antialiased=False)
+
+    # Make lists to hold the new ground truths, tracks, detections, and clutter
+    new_truths, new_tracks, new_measurements, new_clutter = [], [], [], []
+    for truth in truths_by_time[i]:
+        new_truths.append([truth.state_vector[0], truth.state_vector[2]])
+    for state in tracks_by_time[i]:
+        new_tracks.append([state.state_vector[0], state.state_vector[2]])
+    for measurement in all_measurements[i]:
+        if type(measurement) == TrueDetection:
+            new_measurements.append([measurement.state_vector[0], measurement.state_vector[1]])
+        elif type(measurement == Clutter):
+            new_clutter.append([measurement.state_vector[0], measurement.state_vector[1]])
+
+    # Plot the contents of these lists on the right axis
+    if len(new_truths) > 0: truths.set_offsets(new_truths)
+    if len(new_tracks) > 0: tracks.set_offsets(new_tracks)
+    if len(new_measurements) > 0: measurements.set_offsets(new_measurements)
+    if len(new_clutter) > 0: clutter.set_offsets(new_clutter)
+
+    # Create a legend. The use of Line2D is purely for the visual in the legend
+    data_types = [Line2D([0], [0], color='white', marker='o', markerfacecolor='blue', markersize=15,
+                         label='Ground Truth'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='orange', markersize=15,
+                         label='Clutter'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='green', markersize=15,
+                         label='Detection'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='red', markersize=15,
+                         label='Track')]
+    axR.legend(handles=data_types)
+
+    return sf, truths, tracks, measurements, clutter
+
+# Set up the x, y, and z space for the 3D axis (the left one)
+xx = np.linspace(x_min-5, x_max+5, 100)
+yy = np.linspace(y_min-5, y_max+5, 100)
+x, y = np.meshgrid(xx,yy)
```

Comment:
```suggestion
x, y = np.meshgrid(xx, yy)
```
Missing whitespace after comma